qiskit_optimization/algorithms/grover_optimizer.py

```diff
@@ -236,6 +235,21 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
                     logger.info('Current Optimum Value: %s', optimum_value)
                     improvement_found = True
                     threshold = optimum_value
+
+                    # trace out work qubits and store samples
```

Comment:
Is this change also related to bit ordering?

Reply:
This comes from [#96](
https://github.com/Qiskit/qiskit-optimization/pull/96/files) because I expected this PR to be merged after #96 since 
the unittest of `GroverOptimizer` was failing and CI was not working. I think we'd better merge #96 before this PR since `GroverOptimizer` also relies on `generate_solution`.